fixieai/agents-ts/package.json

```diff
@@ -1,20 +1,22 @@
 {
-  "name": "@fixie/sdk",
-  "private": true,
-  "version": "1.0.0",
+  "name": "@fixieai/sdk",
+  "version": "1.0.0-6",
   "description": "The Fixie SDK (https://fixie.ai/)",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "files": [
     "/dist",
     "/src"
   ],
+  "bin": {
+    "serve-bin": "dist/serve-bin.js"
+  },
   "scripts": {
     "lint": "eslint . --max-warnings 0",
     "lint:fix": "eslint . --fix",
     "format": "dprint fmt",
     "format:check": "dprint check",
-    "postinstall": "npm run build",
+    "prepublishOnly": "npm run build",
```

Comment:
This was a mistake – by putting it as `postinstall`, I had consumers running `build`, which is not what we want.